ibl_pipeline/behavior.py

```diff
@@ -372,28 +372,37 @@ class TrialSet(dj.Imported):
     """
 
     # Knowledge based hack to be formalized better later
-    key_source = CompleteTrialSession & 'trial_session_complete = 1'
+    key_source = CompleteTrialSession & 'trial_session_complete=1'
 
     def make(self, key):
         trial_key = key.copy()
         # excluded_trial_key = key.copy()      
         eID = (acquisition.Session & key).fetch1('session_uuid')
 
-        trials_feedback_times, trials_feedback_types, trials_gocue_times, \
-            trials_intervals, trials_rep_num, trials_response_choice, trials_response_times, \
+        trials_feedback_times, trials_feedback_types, trials_intervals, \
+            trials_rep_num, trials_response_choice, trials_response_times, \
             trials_contrast_left, trials_contrast_right, \
-            trials_visual_stim_times, trials_included, \
-            trials_p_left = \
+            trials_included, trials_p_left = \
             ONE().load(eID, dataset_types=['_ibl_trials.feedback_times', '_ibl_trials.feedbackType',
-                                           '_ibl_trials.goCue_times', '_ibl_trials.intervals', '_ibl_trials.repNum',
+                                           '_ibl_trials.intervals', '_ibl_trials.repNum',
                                            '_ibl_trials.choice', '_ibl_trials.response_times',
                                            '_ibl_trials.contrastLeft', '_ibl_trials.contrastRight',
-                                           '_ibl_trials.stimOn_times', '_ibl_trials.included',
-                                           '_ibl_trials.probabilityLeft'])
-
+                                           '_ibl_trials.included', '_ibl_trials.probabilityLeft'])
+        if key['lab_name'] == 'wittenlab':
+            trials_visual_stim_times = np.squeeze(ONE().load(eID, dataset_types='_ibl_trials.stimOn_times', clobber=True))
+        else:
+            trials_visual_stim_times = ONE().load(eID, dataset_types='_ibl_trials.stimOn_times')
+
+        if len(trials_visual_stim_times) == 1:
+            trials_visual_stim_times = np.squeeze(trials_visual_stim_times)
+        
+        # for debugging purpose
+        # print(key['session_start_time'])
+        # print(len(trials_feedback_times), len(trials_feedback_types), len(trials_intervals), \
+        #     len(trials_rep_num), len(trials_response_choice), len(trials_response_times), len(trials_contrast_left), \
+        #     len(trials_contrast_right), len(trials_visual_stim_times), len(trials_included), len(trials_p_left))
```

Comment:
^ this would be a good place to use logging functions - but not a blocker for now, lets leave as is and discuss for ongoing improvement.